agent_issuance/src/credential/aggregate.rs

```diff
@@ -304,12 +369,31 @@ impl Aggregate for Credential {
     }
 }
 
+fn calculate_expiration_timestamp(overwrite: Option<CredentialExpiry>) -> Option<identity_core::common::Timestamp> {
+    #[cfg(feature = "test_utils")]
+    let now = "2010-01-01T00:00:00Z".parse::<chrono::DateTime<chrono::Utc>>().unwrap();
+    #[cfg(not(feature = "test_utils"))]
+    let now = chrono::Utc::now();
+
+    let config = config().credential_expiry.clone();
+
+    match overwrite.unwrap_or(config) {
+        CredentialExpiry::Relative(duration) => {
+            let expires_at = now + duration.to_chrono_at_datetime(now);
+            Some(identity_core::common::Timestamp::from_unix(expires_at.timestamp()).unwrap())
```

Comment:
Please handle error since this can panic:
```rust
#[test]
fn large_relative_expiry_panics_due_to_invalid_timestamp() {
    let overwrite = Some(CredentialExpiry::Relative(
        iso8601_duration::Duration::from_str("P10000000Y").unwrap(), // <-- seven 0'
    ));

    calculate_expiration_timestamp(overwrite);
    // thread 'credential::aggregate::test' panicked at agent_issuance/src/credential/aggregate.rs:385:86:
    // called `Result::unwrap()` on an `Err` value: InvalidTimestamp(ComponentRange(ComponentRange { name: "timestamp", minimum: -377705116800, maximum: 253402300799, value: 316622289664, conditional_range: false }))
}
```